base/show.jl

```diff
@@ -2105,33 +2107,44 @@ function show_tuple_as_call(io::IO, name::Symbol, sig::Type, demangle=false, kwa
     sig = (sig::DataType).parameters
     show_signature_function(env_io, sig[1], demangle)
     first = true
-    print_within_stacktrace(io, "(", color=:light_black)
+    print(io, "(")
```

Comment:
Of course now I notice this should be bold, to match the closing bracket:
```
print_within_stacktrace(io, "(", bold=true)
```

Reply:
Would you mind PRing that?

Reply:
Will do. May give it a day in case other mistakes surface?

Reply:
Done, I just made #38129.